EosPayload/lib/position_aware_driver_base.py

```diff
@@ -0,0 +1,31 @@
+from abc import ABC
+
+import EosLib.packet.packet
+from EosLib import Device, Type
+from EosLib.format.position import Position
+
+from EosPayload.lib.driver_base import DriverBase
+from EosPayload.lib.mqtt import Topic
+
+
+class PositionAwareDriverBase(DriverBase, ABC):
+
+    # TODO: Move everything out of init
+    def __init__(self, output_directory: str):
+        super().__init__(output_directory)
+        self.latest_position = Position()
+
+    def setup(self) -> None:
+        self._mqtt.register_subscriber(Topic.POSITION_UPDATE, self.position_callback)
```

Comment:
call super here, even though right now it doesn't do anything

Reply:
Done